set-matter.sh

```diff
@@ -18,6 +18,11 @@ case $key in
     shift # past argument
     shift # past value
     ;;
+    -f|--font)
```

Comment:
I guess since this is written in a case switch so we cannot have both the options `-p` and `-f` working together.

Reply:
This is actually fine. If you want to know why i suggest you to read the documentation for shift command [here](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_09_07.html) :). What is happening in this while loop is that it checks for all flags attached to the command and processes them one by one.